.github/workflows/pypi.yml

```diff
@@ -18,27 +18,43 @@ concurrency:
 
 jobs:
   build:
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
I think we were only testing earliest and latest release. Adding 2 extra ones may be too much woth our very long tests.

Reply:
At this point we can delegate the compilation to user or precompile on our side:

What is included in the matrix is precompiled, the reset is compiled from the source dist.

The most flexible and realistic use-case for us is to delegate everything to user and supply pypi only with sdist, not bdist

Reply:
Anyway the user has to compile some C code or scan at some point

Reply:
> I think we were only testing earliest and latest release. Adding 2 extra ones may be too much worth our very long tests.

This only tests you can import pytensor + builds linux wheels